src/app/components/applications/SSV/MyAccount/components/SingleOperator/SingleOperator.tsx

```diff
@@ -111,6 +111,10 @@ const SingleOperator = () => {
     window.open(`${beaconchaBaseUrl}/validator/${publicKey}/?version=${config.links.EXPLORER_VERSION}&network=${config.links.EXPLORER_NETWORK}`, '_blank');
   };
 
+    const sortValidatorsByStatus = () => {
+        setOperatorsValidators(prevState => [...prevState.sort((a: any, b: any) => a.status === b.status ? 0 : a.status ? -1 : 1)]);
+    };
```

Comment:
This code block doesn't have proper indent

Reply:
done